src/PointwiseFunctions/AnalyticSolutions/Burgers/Linear.hpp

```diff
@@ -36,8 +36,8 @@ class Linear {
   static constexpr OptionString help{"A spatially linear solution"};
 
   Linear() = default;
-  Linear(const Linear&) noexcept = delete;
-  Linear& operator=(const Linear&) noexcept = delete;
+  Linear(const Linear&) noexcept = default;
```

Comment:
Why do you need these constructors?

Reply:
I needed them in an early version of these commits.  I can get rid of them.  (The constructor is used to initialize the global cache in the test, but I don't need to keep a copy of the analytic solution for my own use anymore so I can change that to a move.)